app/src/main/AndroidManifest.xml

```diff
@@ -1,56 +1,50 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest
```

Comment:
Can you undo all your AndroidManifest changes minus the new splash activity? There's a lot of new labels on activities and stuff too; we need to keep this code review focused.